### ISSUE TITLE
React webpack issue

### ISSUE THREAD

**Issue by vadim312** [vadim312/React-CRUD-TS#1]
I am creating this as I am getting webpack issue

**Comment by darrel0529** [comment]
Please tell me your webpack issue.

**Comment by vadim312** [comment]
I just wanted to know its possibility for me to get house if something new street is built

**Comment by darrel0529** [comment]
Yeah, it's possible to get house (3 rooms) in TongIL street, but don't want to do it. So waiting for new street. Said that it will be possible if built, but still not sure.

**Comment by vadim312** [comment]
Aha, so is there any guy to get it even TongIL st?

**Comment by vadim312** [comment]
[tbtc-source.zip](https://github.com/vadim312/React-CRUD-TS/files/14685774/tbtc-source.zip)

@fanghuateng0621

**Comment by vadim312** [comment]
[tbtc-js-sdk.zip](https://github.com/vadim312/React-CRUD-TS/files/14685831/tbtc-js-sdk.zip)
